netbench/netbench-driver/etc/Dockerfile

```diff
@@ -73,10 +73,20 @@ RUN set -eux; \
 
 ARG DRIVER="s2n-quic"
 ARG ENDPOINT="client"
+ENV ENDPOINT_ENV=$ENDPOINT
```

Comment:
Does this get used anywhere? I'm not seeing that it does

Reply:
Nope I was testing something, I believe I removed it in my newest commit